dash_table/Format.py

```diff
@@ -0,0 +1,270 @@
+import inspect
+import collections
+
+
+def get_named_tuple(name: str, dict: dict):
+    return collections.namedtuple(name, dict.keys())(*dict.values())
+
+
+Align = get_named_tuple('align', {
+    'default': '',
+    'left': '<',
+    'right': '>',
+    'center': '^',
+    "right_sign": '='
+})
```

Comment:
Not sure if this is better than the Enum approach but it does seem more flexible / lightweight from the user's perspective

Reply:
Looks good to me! It has all the right behaviors AFAICT and seems easier for us to work with (`value not in Align`) than a straight `class Align(object)`.